profiles/opentelemetry_dev/Containerfile

```diff
@@ -0,0 +1,24 @@
+FROM ghcr.io/pulp/pulp-ci-centos:latest
```

Comment:
if you really need to modify the base image, you can do `FROM localhost/oci_env/pulp:base` and then change the pulp services to use your new image

Reply:
Here's an example of how to use a custom base image in the compose.yaml file

```
services:
  pulp:
    build:
      context: /path/to/my/profile/
```